doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -3252,6 +3254,54 @@ XROAD_PROXY_UI_API_PARAMS
 XROAD_SIGNER_CONSOLE_PARAMS
 ```
 
+### 21.1 Updating Proxy Service's memory allocation command line arguments
```

Comment:
How about the Security Server Sidecar and dockerized environments? Is the helper script usable for them as well or should there be a note that the script is intended for native deployments only?

Reply:
It should work also in containerized  environments, added reference in sidecar documentation about usage of new script